melodic/distribution.yaml

```diff
@@ -2605,6 +2605,16 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/dynamixel-workbench-msgs.git
       version: melodic-devel
     status: developed
+  dynamixel_interface:
+    doc:
+      type: git
+      url: https://github.com/csiro-robotics/dynamixel_interface.git
+      version: noetic-devel
```

Comment:
@TJMolnar `noetic-devel` for Melodic?

Reply:
@hidmic I have updated the noetic-devel branch to be targeting c++14 explicitly. The branch is specifically compatible with both noetic and melodic so I went with the latest version as a branch name, please advise if a different convention is preferred.